### DISCUSSION TITLE
Anyway to test out future releases prior to completion?

### DISCUSSION THREAD

**Discussion by sf-walsh** [aws-solutions/workload-discovery-on-aws#429] [General]
Interested in testing the solution within my organization but I would need to deploy within an existing VPC and account. 

I can see from the roadmap and milestones that the feature which is complete [#169](https://github.com/aws-solutions/workload-discovery-on-aws/issues/169) but wouldn't be available until the 2.1 release which is currently several backlog items away.

Is there any way a beta branch for those that are willing to test this code out prior to deployment can be made available, with the caveat of course with all code that it is use at your own risk? Would also provide feedback or fixes should any issues be encountered along the way if possible.

Thanks in advance.

**Comment by svozza** [comment]
Hi there, apologies for the late reply! Unfortunately we don't have an external beta program but we did released v2.1.0 last month, which has the feature you are looking for. Be sure to verify that the VPC you are deploying the solution to has the necessary configuration to allow Workload Discovery to function as per the [documentation](https://docs.aws.amazon.com/solutions/latest/workload-discovery-on-aws/deployment-process-overview.html).

**Comment by aws-ymson** [comment]
Closing this out: Workload Discovery on AWS has been deprecated as of 08/14/2026 and is no longer being maintained.
